SETUP/INSTALL.md

```diff
@@ -16,8 +16,8 @@ The following lists supported versions for the four primary middleware
 components.
 
 ### PHP
-PHP version 7.4 is the minimum supported version. Limited testing has been done
-on PHP 8.1 and 8.3.
+PHP version 8.1 is the minimum supported version. Limited testing has been done
+on PHP 8.3.
```

Comment:
When we upgrade to Ubuntu 24.04 we'll be on PHP 8.3 and that will have lots of testing so this is probably better phrased:
```suggestion
PHP version 8.1 is the minimum supported version although limited testing has been done
on PHP versions < 8.3.
```